pgbouncer/manifest.json

```diff
@@ -10,6 +10,7 @@
     "linux",
     "mac_os"
   ],
+  "package_deps": false,
```

Comment:
Actually, no, we don't want it to be mandatory! We want it to be an optional field, that's what @olivielpeau and I decided the other day when we talked about it.